pkg/kotsadm/kotsadm_objects.go

```diff
@@ -575,3 +577,55 @@ func kotsadmService(namespace string) *corev1.Service {
 
 	return service
 }
+
+func kotsadmIngress(namespace string, config ingresstypes.Config) *extensionsv1beta1.Ingress {
+	kotsadmPath := "/kotsadm"
+	if config.KotsadmPath != "" {
+		kotsadmPath = config.KotsadmPath
+	}
+
+	ingress := &extensionsv1beta1.Ingress{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1beta1",
+			Kind:       "Ingress",
+		},
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
we should allow for arbitrary annotations. its often the case that cloud providers support ingress through annotations